extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -9,18 +9,21 @@ import * as dataSources from '../models/dataSources/dataSources';
 import * as utils from '../common/utils';
```

Comment:
please reorder the import statements, there is a vscode extension that does the reordering: vsc-organize-imports